12LessonTwelve/homework/css/style.css

```diff
@@ -0,0 +1,143 @@
+h1{
+  text-align:center;
+  font-size: 1.5em;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.container
+{
+  margin: auto;
+  font-family: sans-serif;
+}
+
+section
+{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+article
+{
+  margin: 10px;
+  width: 320px;
```

Comment:
A good width for smaller screen sizes but consider using a responsive (%) width for larger devices. Since you have 4 items, to get them side by side you'll want ~25% (potentially a little smaller due to margin and padding).